qcfractal/interface/collections/torsiondrive_dataset.py

```diff
@@ -324,5 +326,76 @@ def visualize(self,
 
         return custom_plot(traces, custom_layout, return_figure=return_figure)
 
+    def status(self,
+               specs: Union[str, List[str]] = None,
+               collapse: bool = True,
+               status: Optional[str] = None,
+               detail: bool = False) -> 'DataFrame':
+        """Returns the status of all current specifications.
+
+        Parameters
+        ----------
+        specs : Union[str, List[str]], optional
+            Description
+        collapse : bool, optional
+            Collapse the status into summaries per specification or not.
+        status : Optional[str], optional
+            If not None, only returns results that match the provided status.
+        detail : bool, optional
+            Shows a detailed description of the current status of incomplete jobs.
+
+        Returns
+        -------
+        DataFrame
+            A DataFrame of all known statuses
+        """
+
+        if detail is False:
+            return super().status(specs, collapse=collapse, status=status)
+
+        if status not in [None, 'INCOMPLETE']:
+            raise KeyError("Detailed status is only available for incomplete procedures.")
+
+        if not (isinstance(specs, str) or len(specs) == 1):
+            raise KeyError("Detailed status is only available for a single specification at a time.")
+
+        mapper = self._get_procedure_ids(specs)
+        reverse_map = {v: k for k, v in mapper.items()}
```

Comment:
Can this ever result in a double up? Or is it because each service has to be unique anyways (as it would call the already finished one from the database)?

Reply:
Your right, this could potentially result in a doubling if someone accidentally had two of the same starting configurations.

Reply:
I think we might go with it, the only thing it hits is the labels. You still get the current status of all running computations. Fixing it would be difficult.